jupyter_server/services/sessions/handlers.py

```diff
@@ -77,6 +77,7 @@ async def post(self):
         kernel = model.get("kernel", {})
         kernel_name = kernel.get("name", None)
         kernel_id = kernel.get("id", None)
+        custom_env_vars = kernel.get("env", {})
```

Comment:
Shouldn't this be conditional based on the value of `allow_setup_custom_env_variables`?

Reply:
Dear @Zsailer I think it is a good idea. Sure, I agree. Thank you for suggestion.